### PR TITLE
StackRox: Re-enable nightly with ebpf for s390x

### PR DESCRIPTION
**Pull request by dale-fu**
Re-enabling StackRox e2e testing with ebpf on s390x.

cc: @gbhure

### CODE REVIEW

**Review by gbhure — APPROVED** [review]

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml [hunk 1]
```diff
@@ -211,6 +211,13 @@ tests:
       COLLECTION_METHOD: core_bpf
     workflow: stackrox-automation-flavors-ibmcloudz-e2e
   timeout: 5h0m0s
+- as: nightly-ibmcloudz-ebpf-qa-e2e-tests
+  cron: 0 7 * * *
```
Comment (msugakov) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:215]: I think we have periodic jobs in `stackrox-stackrox-nightlies.yaml` file, not in `stackrox-stackrox-master.yaml` but I'd defer to @gavin-stackrox to confirm.

**Comment by dale-fu** [comment]
Might close this ticket as it is a duplicate of https://github.com/openshift/release/pull/44286/.

**Pull request closed without merging**